src/auth0-sso-login.js

```diff
@@ -170,16 +170,16 @@ export default class auth {
               });
             });
         } else {
-          reject({ error: 'no_token_available', errorDescription: 'Failed to get valid token.', authResultError: authResult.error });
+          reject({ error: 'no_token_available', errorDescription: 'Failed to get valid token.', authResultError: authResult ? authResult.error : undefined });
         }
       });
     })
-      .catch((error) => {
-        if (retries < 4 && error.authResultError === undefined) {
-          return new Promise(resolve => setTimeout(() => resolve(), 1000))
-            .then(() => this.renewAuth(retries + 1));
-        }
-        throw error;
-      });
+    .catch((error) => {
```

Comment:
This violates the eslint rules of this library.

Reply:
Hmmm, true.  Fixed.